### PR TITLE
docs: reorganise

### PR DESCRIPTION
**Pull request by bodinsamuel**
- Split Content
- Add more answers about extraction

### CODE REVIEW

File: docs/Extraction.md [hunk 1]
```diff
@@ -114,18 +130,79 @@ For this URL: <https://www.algolia.com/products/crawler/>
...
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+```
+
+### Using Canonical
+
+You can redirect the crawl to an other page and indirectly exclude this page from the crawl by using the canonical meta tag.
+It is also useful to ignore **query params**, e.g: pagination, search term, etc...
+Learn more about [canonical meta tag](https://ahrefs.com/blog/canonical-tags/).
```
Comment (sbellone) [inline docs/Extraction.md:192]: For consistency with other links:
```suggestion
Learn more about [canonical meta tag](https://support.google.com/webmasters/answer/139066?hl=en).
```

File: docs/Extraction.md [hunk 2]
```diff
@@ -114,18 +130,79 @@ For this URL: <https://www.algolia.com/products/crawler/>
...
+```html
+<head>
+  <link rel=“canonical” href=“/another-page.html” />
+</head>
+```
+
+## Executing Javascript
+
+The Algolia Crawler supports JavaScript-rendered websites, but we don't yet expose this option in the plugin.
+Please [contact us](mailto:support@algolia.com) to enable it.
```
Comment (sbellone) [inline docs/Extraction.md:203]: Are we ready for that? :)

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# FAQ
+
+Here is general purpose FAQ. If you are looking for technical answer, please go to our [Technical documentation](Extraction.md).
```
Comment (sbellone) [inline docs/FAQ.md:3]:
```suggestion
Here is general purpose FAQ. If you are looking for technical answers, please go to our [Technical documentation](Extraction.md).
```

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+You can configure notifications in the [account settings](https://crawler.algolia.com/admin/user/settings/) of your Crawler Admin Console.
+
+## My website is not crawled<!-- omit in toc -->
+
+The monitoring tab in the Crawler UI will provide information to help you improve the crawler behavior.
+By default we respect most of the standard web rules: robots.txt, password protection, canonical, redirection, etc...
+
+You can help the crawler by:
+
+- Allowing our user-agent in [your robots.txt]([<!-- omit in toc -->](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-ip-address-can-i-use-for-ip-whitelisting))
```
Comment (sbellone) [inline docs/FAQ.md:28]:
```suggestion
- Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-is-the-user-agent-of-the-crawler-useful-for-whitelisting))
```

File: docs/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+The plugin currently only support custom domains that are configured in Netlify (**Settings > Domain management > Custom domains**).
+
+## Do I need to display the Algolia logo?
+
+Yes, but only if you are on a free plan.
+Please see our official answer [here](https://www.algolia.com/doc/faq/accounts-billing/do-i-need-to-display-the-algolia-logo-when-i-am-on-the-free-plan/).
+
+## I am an Algolia paying customer, can I use my own Application?
+
+Yes, but it currently require a manual intervention.
```
Comment (sbellone) [inline docs/FAQ.md:42]:
```suggestion
Yes, but it currently requires a manual intervention.
```

File: docs/GettingStarted.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+Once the crawl is done, you can check your Algolia index, which contains the extracted records. We apply a standard relevance configuration by default, but you can fine-tune it as you want in the index settings.
+
+> Find out [what's inside your records](Extraction.md).
+
+<img src="/docs/screenshots/algolia-index.png?raw=true" alt="Your Algolia Index.">
+
+### Installing the front-end bundle
+
+> We recommend using our bundle as it's compatible out of the box with the records we extract from your site.
+> You can still [build your custom UI](#can-i-build-my-own-ui) if you want to.
```
Comment (sbellone) [inline docs/GettingStarted.md:68]: The link needs to be updated

File: docs/GettingStarted.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+You can find an HTML code snippet in the [Crawler Admin Console](https://crawler.algolia.com/admin/netlify) that you can use in your code. Make sure to replace the variables with the provided ones:
+
+- `YOUR_ALGOLIA_APP_ID`: the unique identifier of your Algolia application
+- `YOUR_ALGOLIA_API_KEY`: your Algolia search-only API key (you can find it on your [Algolia dashboard](https://www.algolia.com/api-keys))
+- `YOUR_NETLIFY_SITE_ID`: the unique identifier of your Netlify site (we prefill this for you in the snippet from the [Crawler Admin Console](https://crawler.algolia.com/admin/netlify))
+- `YOUR_TARGET_GIT_BRANCH`: Target git branch, either a fixed one (e.g. `'master'`) or a dynamic one using `process.env.HEAD`. See ["Using Multiple branches"](./frontend#using-multiple-branches) of the frontend doc.
```
Comment (sbellone) [inline docs/GettingStarted.md:75]: The link needs to be updated

File: docs/GettingStarted.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+### Indexing
+
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+When it receives a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
+
+You can find information about your current crawler in the [Netlify deploy logs](https://docs.netlify.com/monitor-sites/logs/#deploy-log).
+
+> Note: by default, we only build the `master` branch. We can however create one crawler (targeting one Algolia index) per Git branch, so that you can have, for example, a production index on `master` and development index on `develop`. You need to configure the [`branches` plugin input](./plugin#available-parameters) to enable this feature.
+> If you're using our front-end library, you'll also need to [pass the correct branch](./frontend#using-multiple-branches) to the library parameters.
```
Comment (sbellone) [inline docs/GettingStarted.md:51]: The link needs to be updated

File: docs/GettingStarted.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+### Indexing
+
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+When it receives a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
+
+You can find information about your current crawler in the [Netlify deploy logs](https://docs.netlify.com/monitor-sites/logs/#deploy-log).
+
+> Note: by default, we only build the `master` branch. We can however create one crawler (targeting one Algolia index) per Git branch, so that you can have, for example, a production index on `master` and development index on `develop`. You need to configure the [`branches` plugin input](./plugin#available-parameters) to enable this feature.
```
Comment (sbellone) [inline docs/GettingStarted.md:50]: The link needs to be updated

File: docs/Extraction.md (hunk 2 above)
Comment (bodinsamuel) [reply docs/Extraction.md:203]: the text was already there, just copied from FAQ :p

File: docs/GettingStarted.md (hunk 6 above)
Comment (bodinsamuel) [reply docs/GettingStarted.md:68]: ah good catch

File: docs/GettingStarted.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+### Indexing
+
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+When it receives a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
+
+You can find information about your current crawler in the [Netlify deploy logs](https://docs.netlify.com/monitor-sites/logs/#deploy-log).
+
+> Note: by default, we only build the `master` branch. We can however create one crawler (targeting one Algolia index) per Git branch, so that you can have, for example, a production index on `master` and development index on `develop`. You need to configure the [`branches` plugin input](../plugin#available-parameters) to enable this feature.
```
Comment (sbellone) [inline docs/GettingStarted.md:50]: Still not good

File: docs/GettingStarted.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+> If you're using our front-end library, you'll also need to [pass the correct branch](../frontend#using-multiple-branches) to the library parameters.
```
Comment (sbellone) [inline docs/GettingStarted.md:51]: Still not good

File: docs/FAQ.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
+# FAQ
+
+Here is general purpose FAQ. If you are looking for technical answers, please go to our [Technical documentation](Extraction.md).
```
Comment (sbellone) [inline docs/FAQ.md:3]:
```suggestion
This is a general purpose FAQ. If you are looking for technical answers, go to our [Technical documentation](Extraction.md).
```

File: docs/FAQ.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+You can configure notifications in the [account settings](https://crawler.algolia.com/admin/user/settings/) of your Crawler Admin Console.
+
+## My website is not crawled<!-- omit in toc -->
+
+The monitoring tab in the Crawler UI will provide information to help you improve the crawler behavior.
+By default we respect most of the standard web rules: robots.txt, password protection, canonical, redirection, etc...
+
+You can help the crawler by:
+
+- Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-ip-address-can-i-use-for-ip-whitelisting)
```
Comment (sbellone) [inline docs/FAQ.md:28]:
```suggestion
- Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-is-the-user-agent-of-the-crawler-useful-for-whitelisting)
```

File: docs/GettingStarted.md (hunk 11 above)
Comment (bodinsamuel) [reply docs/GettingStarted.md:51]: works for me, what does not work for you?

Comment (sbellone) [reply docs/GettingStarted.md:51]: Indeed it's the one below sorry: `Using Multiple branches`

Comment (bodinsamuel) [reply docs/GettingStarted.md:51]: arg yes

**Review by sbellone — APPROVED** [review]
👍

**Pull request merged**